app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ArticlesController < ApplicationController
     def index
         @articles = Article.all
+        @category_news = NewsFeedService.get_feed('all', 'cnn', 'http://us.cnn.com', 4) 
```

Comment:
Change the variable name to explain what is actually stored in it better. Something like `@external_articles`.

Reply:
done